lifechanger/src/app/header/header.component.html

```diff
@@ -14,25 +14,25 @@
         <div>
           <ul class="header__nav-links">
             <li>
-              <a routerLink="/home" class="header__nav-link">Home</a>
+              <a [routerLink]="['/']" class="header__nav-link">Home</a>
```

Comment:
i think it make more sense to have '/home' not just empty route

Reply:
Dear Mentor, thank you for reviewing the code.
I repaired the home **routing** according to the provided remark.